html/changelogs/Kurfurst.yml

```diff
@@ -3,4 +3,6 @@ delete-after: True
 changes:
  - tweak: Battering rams now properly cancel when you move away from the target door
  - tweak: Electrolyzers are now compatible with a much wider variety of reagent_containers, including cans
- - tweak: HE lights now no longer prevent you from downgrading the light fixture to a non-HE version
\ No newline at end of file
+ - tweak: HE lights now no longer prevent you from downgrading the light fixture to a non-HE version

```

Comment:
You should make a new changelog file, because editing an existing one makes it harder to add to the changelog itself.

I've never understood people having a single changelog file, you're going to have more than one PR up at once.